Userland/Libraries/LibDl/dlfcn.cpp

```diff
@@ -60,3 +60,15 @@ void* dlsym(void* handle, const char* symbol_name)
     }
     return result.value();
 }
+
+int dladdr(void* addr, Dl_info* info)
+{
+    auto result = __dladdr(addr, info);
+    if (result.is_error()) {
+        // FIXME: According to the man page glibc does _not_ make the error
+        // available via dlerror(), however we do. Does this break anything?
+        store_error(result.error().text);
+        return 0;
+    }
+    return 1;
+}
```

Comment:
pls configure your editor to put a newline at the end of files :^)

Reply:
Every single time. I blame CLion. :)

Reply:
`File | Settings | Editor --> Ensure blank line before end of file on Save` in case you havent already :^)

Reply:
Neat, let's see if that fixes it in the future. :partying_face: